app/views/static_pages/privacy_policy.html.erb

```diff
@@ -97,6 +97,17 @@
       </li>
     </ul>
 
+    <h2 class="govuk-heading-l">
+      How we access your personal data
+    </h2>
+
+    <p class="govuk-body">
+      To process your claim we will compare the information you have provided with records kept on the Teachers
+      Pension Service and the Database of Qualified Teachers. This is so we can validate your claim, we will not allow
```

Comment:
```suggestion
      Pension Service and the Database of Qualified Teachers. This is so we can validate your claim. we will not allow
```

Reply:
Amended this and added in the ’validate your claim. We will not allow’.